pipeline/transformer/transformer_calm/src/test/scala/uk/ac/wellcome/platform/transformer/calm/NormaliseTextTest.scala

```diff
@@ -0,0 +1,67 @@
+package uk.ac.wellcome.platform.transformer.calm
+
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+
+class NormaliseTextTest extends AnyFunSpec with Matchers {
+
+  it("preserves basic text tags") {
+    val str = "Some text with some <em>basic text tags</em> inside."
+    NormaliseText(str) shouldBe str
+  }
+
+  it("strips non-basic text tags") {
+    val str = "<div>Tags should be <in>removed</in></div>"
+    NormaliseText(str) shouldBe "Tags should be removed"
+  }
+
+  it("removes script tags and inner content") {
+    val str =
+      """Nothing here...<script>window.alert("NASTY");</script> Nothing."""
+    NormaliseText(str) shouldBe "Nothing here... Nothing."
+  }
+
+  it("does not error when missing closing tags") {
+    val str = "Someone forgot <em>to close something.."
+    NormaliseText(str) shouldBe "Someone forgot <em>to close something..</em>"
```

Comment:
Where does the closing tag get put with new lines?

Reply:
I don't think there is a simple answer to that, I think it is dependent on other subsequent opening / closing tags but not entirely sure